src/store/api.ts

```diff
@@ -190,6 +185,32 @@ const getAssociatedUSDCTokenAddressPK = multiAsync(
 	}
 );
 
+const getDepositorLPAssociatedTokenAddress = multiAsync(
+	async (connection: Connection, wallet: Wallet) => {
+		const lpTokenMintPK = await getLPTokenMintPK(connection, wallet);
+		return Token.getAssociatedTokenAddress(
+			ASSOCIATED_TOKEN_PROGRAM_ID,
+			TOKEN_PROGRAM_ID,
+			lpTokenMintPK,
+			wallet.publicKey
+		);
+	}
+);
+
+const getAssociatedLiquidityPoolUSDCTokenAddressPK = multiAsync(
```

Comment:
is this the same account as stored in the global market state?

Reply:
we already have a function to get this account so this one is not needed.
https://github.com/credix-finance/credix-app/pull/40/files#diff-3c19dac050eb3dce614baee16edf7f6b60a73b83307660b84f7650e1347adacdR166